### PR TITLE
Revert "[#41] Update jacoco report"

### PR DESCRIPTION
**Pull request by jean-t86**
Reverts nimblehq/android-templates#41

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,7 +22,7 @@ buildscript {
         glide_version               = '4.8.0'
         gson_version                = '2.4'
 
-        kotlin_version              = '1.3.61'
+        kotlin_version              = '1.3.11'
```
Comment (luongvo) [inline build.gradle:25]: why do we need to downgrade kotlin version?

File: build.gradle [hunk 2]
```diff
@@ -54,9 +49,9 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$build_gradle_version"
+        classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "com.dicedmelon.gradle:jacoco-android:$jacoco_android_plugin_version"
+        classpath 'com.dicedmelon.gradle:jacoco-android:0.1.2'
```
Comment (luongvo) [inline build.gradle:54]: why do we remove version definition for those? ☝️

File: config/jacoco.gradle [hunk 3]
```diff
@@ -1,21 +1,14 @@
 apply plugin: 'jacoco'
 
 jacoco {
-    toolVersion = "$jacoco_core_version"
+    toolVersion = "0.8.3"
```
Comment (luongvo) [inline config/jacoco.gradle:4]: please explain the reason why we have to downgrade this jacoco ver too 🙏 I can't get it from PR description https://github.com/nimblehq/android-templates/pull/41
Also, why do we need to remove ver definition from build.gradle?

File: config/jacoco.gradle [hunk 4]
```diff
@@ -29,39 +22,27 @@ task jacocoTestReport(type: JacocoReport) {
                          'android/**/*.*']
 
     def packagesExcluded = ['com/nimbl3/di/**',
-                            'com/nimbl3/ui/**/di/**',
                             'com/nimbl3/extension/**',
                             'com/nimbl3/**/extension/**']
     // def filesExcluded = [ ]
     // def fileFilter = fileGenerated + packagesExcluded + filesExcluded
```
Comment (luongvo) [inline config/jacoco.gradle:28]: redundant comments, or we could enable them back to keep a common config
```
def filesExcluded = [ ]
def fileFilter = fileGenerated + packagesExcluded + filesExcluded
```

File: config/jacoco.gradle [hunk 5]
```diff
@@ -29,39 +22,27 @@ task jacocoTestReport(type: JacocoReport) {
...
     def fileFilter = fileGenerated + packagesExcluded
 
-    classDirectories.from = fileTree(
-        dir: "$project.rootDir/app/build/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
-        excludes: fileFilter
-    ) + fileTree(
-        dir: "$project.rootDir/data/build/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
+    classDirectories = fileTree(
+        dir: "$buildDir/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
```
Comment (luongvo) [inline config/jacoco.gradle:32]: FYI this part will be changed to `$buildDir/intermediates/javac/stagingDebug/classes` after upgraded to 0.8.5 https://github.com/redplanethotels/rphl-android-app/pull/2109, I guess @jean-t86 and @Tuubz  know this.

**Comment by Wadeewee** [comment]
@jean-t86 Is this old version before you merge right?

**Comment by Tuubz** [comment]
@luongvo 

This PR > https://github.com/nimblehq/android-templates/pull/41 was merged too early as it didn't receive enough approvals.

So what we're trying to do here is to revert the changes first, then once this has been approved and merged, we'll bring back this https://github.com/nimblehq/android-templates/pull/41

Is there a better approach for this?

File: config/jacoco.gradle (hunk 5 above)
Comment (Tuubz) [reply config/jacoco.gradle:32]: Let's not forget to check this when bringing back #41 @Wadeewee 🙏

File: config/jacoco.gradle (hunk 4 above)
Comment (Tuubz) [reply config/jacoco.gradle:28]: I have to agree, it doesn't feel like this comment has any value...

What do you mean with `could enable them back to keep a common config` ?

File: build.gradle (hunk 1 above)
Comment (Tuubz) [reply build.gradle:25]: Just reverting the changes from #41

File: build.gradle (hunk 2 above)
Comment (Tuubz) [reply build.gradle:54]: Just reverting the changes from #41

File: config/jacoco.gradle (hunk 3 above)
Comment (Tuubz) [reply config/jacoco.gradle:4]: Just reverting the changes from #41

File: config/jacoco.gradle (hunk 4 above)
Comment (Tuubz) [reply config/jacoco.gradle:28]: Anyhow, we can keep these changes when bringing back #41 

Please keep this in mind @Wadeewee 🙏

**Comment by luongvo** [comment]
A test comment

**Pull request merged**